test/format/replay.c

```diff
@@ -101,12 +101,12 @@ replay_end_timed_run(void)
 {
     /*
      * We'll post a stop timestamp that all worker threads should abide by. There's a potential race
-     * between when we read the current timestamp and before we publish the stop timestamp. During
-     * that time, other threads could do work and advance the current timestamp, potentially beyond
-     * the intended stop timestamp. We pick a stop timestamp far enough in the future that it's
-     * rather unlikely to happen.
+     * between when we read the current timestamp and before we release write the stop timestamp.
```

Comment:
I don't think in the comment we need to emphasis release write. We should just say before we update the stop timestamp.

Reply:
b06cc6cec